wikipedia_transform/models/open_ai_embedding_model.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+
+from wikipedia_transform.models.types import OpenAiEmbeddingModelName
+
+
+class OpenAiEmbeddingModel(BaseModel):
```

Comment:
Usually with Pydantic you want to inherit your own BaseModel, which inherits from pydantic.BaseModel.